Frameworks/UIKit/UIDatePicker.mm

```diff
@@ -814,6 +813,8 @@ static void resetPickerPositions(UIDatePicker* self) {
  @Status Interoperable
 */
 - (void)setMinimumDate:(NSDate*)date {
+    [date retain];
+    [_minimumDate release];
     _minimumDate = date;
```

Comment:
Yeah I suppose its helpful if I actually read your change description :-p ... I need to check the .h but if we specify getters and setters in the property decl it *should* turn off synthesis barring compiler bugs right?

---
In reply to: [74983076](https://github.com/Microsoft/WinObjC/pull/883#discussion_r74983076) [](ancestors = 74983076)